### DISCUSSION TITLE
VIVE XR ELITE all it is missing is passthrough in SteamVR....

### DISCUSSION THREAD

**Discussion by sitsky** [Rectus/openxr-steamvr-passthrough#9] [Ideas]
...sounds like a job for openxr-steamvr-passthrough?? :) :) 
~S

**Comment by Rectus** [comment]
That all depends on whether HTC is willing to give access to the camera images. I can't find it anymore, but there was an inteview with an engineer from HTC who seemed very reluctant to want to open up access to them. Instead they seem to prefer to make OpenXR extensions for every single passthrough feature.

**Comment by sitsky** [comment]
Do you happen to know where they have these extensions?
I want to work in XR :(
Use something like XROverlay to have screens up, but not in a SteamVR
enviroment, in my actual room...

**Comment by Rectus** [comment]
I have no idea. I haven't even heard anything about passthrough support for it on PC, it's possible that they will only support it on standalone. You would have to ask HTC about it.

**Comment by sitsky** [comment]
I will hit them up about it :)
Back to this layer and a question, why wouldn't it show up in my OpenXR Api
screen in SteamVR?
Do I need any other API for it?

**Comment by Rectus** [comment]
It should show up in the list after you run the setup application. No other software needed. You could also try the OpenXR Explorer utility to see if it is installed.

**Comment by sitsky** [comment]
Hmmm... It's not there.... I'll look for and try the OpenXR Explorer.
Oh, I have moved my Steam folder to the root of the C:\, would that affect
anything?

**Comment by Rectus** [comment]
No, the only thing that matters is that the files in the release archive are in the same place you ran the setup app from. You can run it again, and it should say if it's correctly installed or not.

**Comment by Rectus** [comment]
Here is the interview BTW: https://youtu.be/almTJ6qv9Uc?t=1362

**Comment by sitsky** [comment]
Oh sweet thank you, I will check it out :)
I "poked" HTC, they said they are working on it, and it is semi-priority...
we'll see...
In the meantime, I just put a camera on my monitor facing down,
giving me a top view of my keyboard and showing that in an overlay of
XROverlay hehehehe...
RL hack for a VR world? Talk about transcending worlds! :)

**Comment by sitsky** [comment]
Still half messing with this... Noticed something yesterday to report :) 
In SteamVR environment the layer does not exist as an option in my "loaded extensions", BUT
in All-in-one Sports it does! 
Haven't tried to use it, was in an intense tennis match, but I saw it there! Will attempt to use it and report back...
